Cargo.lock

```diff
@@ -1548,7 +1548,7 @@ checksum = "e19b23d53f35ce9f56aebc7d1bb4e6ac1e9c0db7ac85c8d1760c04379edced37"
 dependencies = [
  "hermit-abi 0.4.0",
  "libc",
- "windows-sys 0.59.0",
+ "windows-sys 0.52.0",
```

Comment:
Is this change expected? There's a similar one at line 4651

Reply:
I was a bit surprised by those changes too.

I reverted them manually, and it doesn't look like cargo tries to downgrade it again, so committed up 97e3f6ef -- thanks for noticing!

Reply:
Yeah cargo does something weird. We never did figure out the borsh dependency version change that was happening in the monorepo. Chido eventually added a step to the version bump that just manually reverts it 🙈

Reply:
Yikes, that's rough :sweat_smile: